src/libServer/EthRpcMethods.cpp

```diff
@@ -664,14 +664,14 @@ std::string EthRpcMethods::GetEthEstimateGas(const Json::Value& json) {
   uint256_t value = 0;
   if (json.isMember("value")) {
     const auto valueStr = json["value"].asString();
-    value = stoull(valueStr.c_str(), nullptr, 0);
+    value = uint256_t{valueStr};
   }
 
   uint256_t gasPrice = GetEthGasPriceNum();
   if (json.isMember("gasPrice")) {
     const auto gasPriceStr = json["gasPrice"].asString();
-    const uint256_t gasPriceNum = stoull(gasPriceStr.c_str(), nullptr, 0);
-    gasPrice = max(gasPrice, gasPriceNum);
+    uint256_t inputGasPrice{gasPriceStr};
```

Comment:
Does this constructor throw?

Reply:
Nope, since we use `unchecked` parameter for template it handles errors as specified in the table:
https://www.boost.org/doc/libs/1_80_0/libs/multiprecision/doc/html/boost_multiprecision/tut/ints/cpp_int.html

Reply:
Ok, seems it throws unfortunately if arbitrary string is provided, will fix it.